ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -290,20 +290,12 @@ void testStreamingIngestFromStream() throws FileNotFoundException {
         }
     }
 
-    private boolean createClients(ConnectionStringBuilder dmCsb, ConnectionStringBuilder engineCsb) {
-        try {
-            IngestClientFactory.createClient(dmCsb);
-            ClientImpl localIngestClient = new ClientImpl(dmCsb);
-//            assertTrue(showDatabasePrincipals(localIngestClient));
-        } catch (URISyntaxException ex) {
-            Assertions.fail("Failed to create ingest client", ex);
-            return false;
-        }
+    private boolean hasAuthentication(ConnectionStringBuilder engineCsb) {
```

Comment:
I'd call this method "canCreateClient"

Reply:
But you also use it

Reply:
for executing commands

Reply:
and what you want to check is if the authentication works